prql-compiler/prqlc/README.md

```diff
@@ -32,3 +40,46 @@ FROM
 WHERE
   has_dog
 ```
+
+PRQL query can be executed with CLI tools working with SQL, such as
+[DuckDB CLI](https://duckdb.org/docs/api/cli.html).
+
+```sh
+$ curl -sL https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv -o penguins.csv
```

Comment:
Simply I used the same file as below. Please replace with a more appropriate file.
https://github.com/PRQL/prql/blob/5c6491dfdd9542bbde9e5efbb12e5e81d860c956/web/book/src/integrations/jupyter.md?plain=1#L29